filters/auth/tokeninfo_test.go

```diff
@@ -397,6 +398,48 @@ func TestOAuth2TokenTimeout(t *testing.T) {
 	}
 }
 
+func TestOAuth2Tokeninfo5xx(t *testing.T) {
+	backend := httptest.NewServer(http.HandlerFunc(func(_ http.ResponseWriter, r *http.Request) {}))
+	defer backend.Close()
+
+	authServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusInternalServerError)
+	}))
+	defer authServer.Close()
+
+	opts := TokeninfoOptions{
+		URL:     authServer.URL + testAuthPath,
+		Timeout: testAuthTimeout,
+	}
+	args := []interface{}{testScope}
+
+	t.Logf("ti.options: %#v", opts)
+
+	spec := newOAuthTokeninfoSpec(filters.OAuthTokeninfoAnyScopeName, opts)
+
+	_, err := spec.CreateFilter(args)
+	require.NoError(t, err, "error creating filter")
+
+	fr := make(filters.Registry)
+	fr.Register(spec)
+	r := &eskip.Route{Filters: []*eskip.Filter{{Name: spec.Name(), Args: args}}, Backend: backend.URL}
```

Comment:
Could use
```go
r := eskip.MustParse(`* -> ...`)
```

Reply:
`MustParse` returns a list of routes

Reply:
proxytest.New supports varargs `proxytest.New(fr, r...)`